app/src/main/java/com/gpayinterns/chat/NewMessageActivity.java

```diff
@@ -181,21 +198,10 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
         super.onActivityResult(requestCode, resultCode, data);
     }
 
-    private void sendNewMessage() throws JSONException
-    {
-        String messageText = messageEditText.getText().toString().trim();
-        if(messageText.equals(""))
-        {
-            hideSoftKeyboard();
-            return;
-        }
-        getChatID();
-    }
 
     private void getChatID() throws JSONException
     {
-        String SAMPLE_CURRENT_USER = "3441453482889885209";
-        String URL = "https://gcp-chat-service.an.r.appspot.com/users/" + SAMPLE_CURRENT_USER+"/chats";
+        String URL = "https://gcp-chat-service.an.r.appspot.com/users/" + currentUser+"/chats";
```

Comment:
same here. The hostname part is hardcoded in multiple places.
If we have to change it for some reason, we will have to change it everywhere.

Reply:
This change has been done on "Android: Encapsulated Server URL Details + view time for messages"